components/docs.html

```diff
@@ -208,6 +208,86 @@ <h3 class="comp-lib-avatar-content_b">Example of Badge on Icon</h3>
                     
 
                     <!-- badge section end -->
+
+                    <!-- alert section start -->
+
+
+                    <div class="Alert common-container" id="comp-alert">
+
+                        <div class="common-wrapper">
+
+                            <h1 class="comp-lib-avatar-color">Alert!</h1>
+                            <div class="comp-lib-avatar-content_a">
+                                Alerts are used to attract user's attention for important information without interrupting the user's flow.
+                            </div>
+                            <div class="alert-container">
+                                <div class="wrap-alert alert-red">
+                                    <p class="content-alert">
+                                        <span class="bg-red bg-color">
+                                            <span class="material-icons alert-icon-style">
+                                                info
+                                            </span>
+                                            <strong>Important Note!</strong>Change and try to submit again.
+                                        </span>
+                                    </p>
+                                </div>
+                                
+                                <div class="wrap-alert alert-blue">
+                                    <p class="content-alert">
+                                        <span class="bg-blue bg-color">
+                                            <span class="material-icons alert-icon-style">
+                                                info
+                                            </span>
+                                            <strong>Important Note!</strong>Change and try to submit again.
+                                        </span>
+                                    </p>
+                                </div>
+                                <div class="wrap-alert alert-green">
+                                    <p class="content-alert">
+                                        <span class="bg-green bg-color">
+                                            <span class="material-icons alert-icon-style">
+                                                info
+                                            </span>
+                                            <strong>Important Note!</strong>Change and try to submit again.
+                                        </span>
```

Comment:
space should be added in here, after "Important Note!"

Reply:
Thanks, changes has been done.